.github/workflows/release-postman.yml

```diff
@@ -1,39 +1,28 @@
 name: Postman Release
 on:
-  schedule:
-    - cron: '0 10 ? * *'
+  pull_request:
+    branches: 
+      - main
+    types:
+      - closed
```

Comment:
does this mean that this action will run after any new PRs that is merged into master?